README.md

```diff
@@ -676,6 +677,36 @@ $ where black
         - Working directory: `$ProjectFileDir$`
 	- Uncheck "Auto-save edited files to trigger the watcher"
 
+
+
+### Wing IDE
+
+1. Install `black`.
+
+```console
+$ pip install black
+```
+
+2. Make sure it runs from the command line, e.g.
+
+```console
+$ black --help
+```
+
+3. In Wing IDE, activate **OS Commands** panel  and define the command  **black** with e.g. the following values, to execute black on the currently selected file, using key binding **F1**:
+
+- right-click on a panel -> Insert Tool -> OS Commands
```

Comment:
'on a panel' ->the Wing docs (e.g. https://wingware.com/doc/custom/gui-layout ) seem to call these "toolboxes".   So this could be: "right-click on the toolbar area of a toolbox".

It could also just be "use the Tools > OS Commands menu selection", which I think will add 'OS Commands' to its default location if it's not already present, and then display it.

Reply:
I also corrected it with these changes. Switched to the more simple description "use the Tools->OS Commands" method.  It will indeed display the 'OS Commands' tool in the default location.